DNN Platform/Modules/DigitalAssets/ClientScripts/dnn.DigitalAssets.js

```diff
@@ -2193,6 +2193,7 @@ dnnModule.digitalAssets = function ($, $find, $telerik, dnnModal) {
             if (totalItems == totalSelectedItems) {
                 $("#dnnModuleDigitalAssetsListViewToolbar input[type=checkbox]", "#" + controls.scopeWrapperId).attr('checked', true);
                 $('#dnnModuleDigitalAssetsListViewToolbar>span.dnnModuleDigitalAssetsListViewToolbarTitle', "#" + controls.scopeWrapperId).text(resources.unselectAll);
+                $('#dnnModuleDigitalAssetsListViewToolbar>span.dnnCheckbox').addClass('dnnCheckbox-checke');
```

Comment:
a typo here shouldn't it be "dnnCheckbox-checked" not "dnnCheckbox-checke" you are missing last letter "d" here,

Reply:
You are right! It is fixed now.